### ISSUE TITLE
Can't reproduce the result

### ISSUE THREAD

**Issue by justopit** [JDAI-CV/CM-NAS#3]
Following your instructions, I ran the experiment on SYSU-MM01  and found it was only 56 of Rank-1 in All-Search, while your paper  is 61

**Comment by northeast1st1bing** [comment]
yes ,its same to me. my Rank-1 result is between 55.8 to 53 on SYSU-MM01 in all-search and 44.5 to 42  in single search.

**Comment by ThomasjonLi** [comment]
I find this problem too.If i use the model trained by our gpu(with author's code, 120epoch),rank-1 in sysu is 52.6%(all-single); but if use the author's model, the result turns to be 62%
